src/App.js

```diff
@@ -1,60 +1,75 @@
-import { useEffect, useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import activities from './resources/schedule.json';
 import dayjs from 'dayjs';
+var utc = require('dayjs/plugin/utc');
+dayjs.extend(utc);
 
 function App() {
   const [today, setToday] = useState([]);
   const [tomorrow, setTomorrow] = useState([]);
+  const [restOfWeek, setRestOfWeek] = useState([]);
   const [nextWeek, setNextWeek] = useState([]);
   const [future, setFuture] = useState([]);
 
-  const todayDate = new Date();
-  let tomorrowDate = new Date();
-  tomorrowDate.setDate(todayDate.getDate() + 1);
-  const thisWeekEnds = new Date();
-  thisWeekEnds.setDate(todayDate.getDate() + 2);
+  //   set today's date
+  let todayDate = dayjs.utc().local().format('YYYY-MM-DD');
+  //   set tomorrow's date
+  let tomorrowDate = dayjs.utc().local().add(1, 'day').format('YYYY-MM-DD');
+  //   set end of this week's date
+  let endOfWeekDate = dayjs.utc().endOf('week').format('YYYY-MM-DD');
+  //   set start of next week's date
+  let startOfNextWeekDate = dayjs
+    .utc()
+    .startOf('week')
+    .add(1, 'week')
+    .format('YYYY-MM-DD');
```

Comment:
You don't need to be using the formatted versions to do comparisons. You can simply use the dates.

dayjs  has utilities to compare dates... whether they're on the same day or after a certain day or on the same week or month, etc.

Reply:
oh ok I didn't know about that